src/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/RequestExecutionQueue.cs

```diff
@@ -382,6 +394,21 @@ private async Task ProcessQueueCoreAsync<TRequest, TResponse>(
         }
     }
 
+    private string TryGetLanguageForRequest(IQueueItem<TRequestContext> work,
+        out Exception? ex)
+    {
+        try
+        {
+            var language = _languageServer.GetLanguageForRequest(work.MethodName, work.SerializedRequest);
+            ex = null;
+            return language;
+        }
+        catch (Exception exception)
```

Comment:
do we want to report this somehow?  it would be nice if we could push on this to get fixed at some point.

Reply:
catching naked exception feels... bad.